client/logger

```diff
@@ -0,0 +1 @@
+../cloud/logger
```

Comment:
Can't we use the logger as a dependency? `@raincatcher/logger@1.0.0` inserted via lerna.

That's what `datasync-cloud` seems to be doing anyway